9 2022 release notes (#79)
ni/install-systemlink-enterprise: release-notes/09-2022/FileIngestionDatabaseRecovery.md

### Cleaning up the service

This step must be completed because Helm does not to remove these resources. If they are not removed the upgraded version of the service will not be affected. All dependencies of the service will be created as needed during the upgrade.

1. Run `kubectl get services | grep fileingestion` (or `Select-String` in powershell) to list all the services attached to `fileingestion`
1. Delete all services that contain `mongodb` in their name.

        Example `kubectl delete service fileingestion-mongodb fileingestion-mongodb-headless`

## Port Forwarding for the upgraded MongoDB instance

**Note** In these steps we cannot use the cannot use the service to connect to the database

1. Find the primary replica by running `kubectl exec -it <replica_name> -- mongo`, which should print something like `rs0:PRIMARY>`.

        Example: `kubectl exec -it fileingestion-mongodb-0 -- mongo`

1. Port-forward the pod locally by running `kubectl port-forward pods/fileingestion-mongodb-0 50000:27017`, replacing the `0` with the index of the primary replica.

## Importing the database backup into the new MongoDB instance

> 1. Import the database contents back by running `mongoimport --uri="mongodb://<root_user>:<root_password>@localhost:<port>/files?authSource=admin" --collection=files_metadata --file=<path>`

Reviewer: ```suggestion
1. Run the following command to import the database contents.

`mongoimport --uri="mongodb://<root_user>:<root_password>@localhost:<port>/files?authSource=admin" --collection=files_metadata --file=<path>`

```

>     **Note** `<root_user>` is should be replaced by the root user, `<root_password>` by the root password, `<path>` to the path on your local computer containing the backup, and `<port>` by the port specified on the previous step.

Reviewer: ```suggestion
Where
-  `<root_user>` is the root user
- `<root_password>` is the root password
- `<path>` is the path on your local computer containing the backup
- `<port>` is the port specified in the previous step.

```
